templates/fleet-scheduler-cronjob.yaml

```diff
@@ -0,0 +1,51 @@
+{{ if .Values.fleetscheduler.enable }}
+{{ range $job := .Values.fleetscheduler.jobs }}
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ $job.name }}
+  labels:
+    k8s-service: fleet-scheduler
+spec:
+  {{- with $job.ttl }}
+  ttlSecondsAfterFinished: {{ . }}
+  {{- end}}
+  {{- with $job.deadline }}
+  startingDeadline: {{ . }}
+  {{- end}}
+  schedule: {{ $job.schedule | quote }}
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          restartPolicy: {{ $job.restartPolicy }}
+          volumes:
+            - name: config-volume
+              configMap:
+                name: fleet-scheduler-config
+          containers:
+          - name: {{ $job.name }}
+            image: {{ $job.image }}:{{ $job.tag }}
+            imagePullPolicy: {{ $job.imagePullPolicy }}
+            {{- with $job.command }}
+            command:
+{{ toYaml . | indent 12 }}
+              {{- end }}
+            volumeMounts:
+              - name: config-volume
+                mountPath: /etc/fleet-scheduler
+                readOnly: true
+            env:
+              - name: FLEET_SCHEDULER_CONFIG
+                value: "/etc/fleet-scheduler/config.yaml"
+            resources:
```

Comment:
for the sandbox lets not set these limits, or set them to a lower value so that its able to run on our wee laptops